### PR TITLE
Thai: add fansub data from menome.in.th

### PR DESCRIPTION
**Pull request by pureexe**
[Menome.in.th](http://www.menome.in.th) is Thai anime fan Webboard launch on early 2011 and shutdown on 30 October 2013. However, they were released data dump at https://archive.org/details/menome

So, I want to merge fansub data from Menome into this project.
- Best regrad.

### CODE REVIEW

**Comment by FansubDB** [comment]
I'll check that later, thank you very much for that!

**Comment by FansubDB** [comment]
For next time, can you reindent the json files using http://jsonlint.com/ please?

File: th/2007/winter.json [hunk 1]
```diff
@@ -1,76 +1,52 @@
...
+			"status": "release",
+			"detail": [{
+				"name": "Endless-FS"
+			}, {
+				"name": "OxygenASz-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "OxygenASz-FS"
```
Comment (FansubDB) [inline th/2007/winter.json]: Can you check that? OxygenASz-FS is doing alone the sub of Minami-ke Okawari, or with Endless-FS?

Comment (pureexe) [inline th/2007/winter.json]: from menome data dump OxygenASz-FS had release Minami-ke Okawari EP.1 without Endless-FS
But in EP.2 (and later)  OxygenASz-FS release with Endless-FS.

So i think it must be OxygenASz-FS & Endless-FS.

File: th/2013/spring.json [hunk 2]
```diff
@@ -0,0 +1,919 @@
...
+			"detail": [{
+				"name": "Artstyle Fansub"
+			}, {
+				"name": "Shoutcaster"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Ayumu-FS"
+			}, {
```
Comment (FansubDB) [inline th/2013/spring.json]: Can you check if "NTCS-Fs" did two co sub at Hayate (one with Ayumu-FS and one with DJ-FS)?

File: th/about.html [hunk 3]
```diff
@@ -59,8 +59,9 @@
 				<h3>แหล่งข้อมูล</h3>
 				<ul>
 					<li>ข้อมูลอนิเมะ : <a href="http://myanimelist.net" target="_blank" title="MyAnimeList.net">MyAnimeList.net</a></li>
-					<li>ข้อมูลรูปภาพ : <a href="http://anilist.co" target="_blank" title="MyAnimeList.net">anilist.co</a></li>
-					<li>ข้อมูลแฟนซับ : <a href="https://www.tafasu.com" target="_blank" title="MyAnimeList.net">TAFASU.com</a></li>
+          <li>ข้อมูลรูปภาพ : <a href="http://anilist.co" target="_blank" title="anilist.co">anilist.co</a></li>
+          <li>ข้อมูลแฟนซับ : <a href="https://www.tafasu.com" target="_blank" title="TAFASU.com">TAFASU.com</a></li>
+          <li>ข้อมูลแฟนซับ : <a href="http://www.menome.in.th" target="_blank" title="menome.in.th">menome.in.th</a></li>
```
Comment (FansubDB) [inline th/about.html]: menome.in.th seems a broken link. Is it useful to put their link? A sentence with "some data based from the old menome.in.th" is not sufficient?

File: th/2007/winter.json (hunk 1 above)
Comment (pureexe) [inline th/2007/winter.json]: solved on 	6bdb7a1

File: th/2011/winter.json [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+		}]
+	}, {
+		"name": "Rinne no Lagrange Season 2 Specials",
+		"image": "https://myanimelist.cdn-dena.com/images/anime/6/60967.jpg",
+		"group": [{
+			"status": "release",
+			"detail": [{
+				"name": "DW"
+			}, {
+				"name": "Lya - Fs"
```
Comment (pureexe) [inline th/2011/winter.json]: solved on f5e12db
DW was co sub with Lya-Fs

File: th/2012/spring.json [hunk 5]
```diff
@@ -0,0 +1,777 @@
...
+		}]
+	}, {
+		"name": "Haiyore! Nyaruko-san",
+		"image": "https://myanimelist.cdn-dena.com/images/anime/6/49081.jpg",
+		"group": [{
+			"status": "release",
+			"detail": [{
+				"name": "ACE-Z"
+			}, {
+				"name": "MunashiiKaze-FS"
```
Comment (pureexe) [inline th/2012/spring.json]: MunashiiKaze-FS was co-sub with ExOtIc
and work on co-sub with ACE-Z on Haiyore! Nyaruko-san F but they have wrong anime tag on menome.in.th

Comment (pureexe) [inline th/2012/spring.json]: solved on ab2445f and 9a8dc48

File: th/2012/spring.json [hunk 6]
```diff
@@ -0,0 +1,777 @@
...
+		"image": "https://myanimelist.cdn-dena.com/images/anime/5/53913.jpg",
+		"group": [{
+			"status": "release",
+			"detail": [{
+				"name": "DW"
+			}, {
+				"name": "Emergency-FS"
+			}]
+		}, {
+			"status": "release",
```
Comment (pureexe) [inline th/2012/spring.json]: from menome data Emergency-FS was alone release until EP.6 and since EP.7 is co-sub with DW
so it must be DW&Emergency-FS

Comment (pureexe) [inline th/2012/spring.json]: solved on 41b8756

File: th/2013/spring.json (hunk 2 above)
Comment (pureexe) [inline th/2013/spring.json]: so weird on this.
it's co-sub NTCS-Fs&DJ-FS on EP1 and EP2
but after that, it was changed to NTCS-Fs&Ayumu-FS 
what should i do for this situation?

File: th/about.html (hunk 3 above)
Comment (pureexe) [inline th/about.html]: solved on 50a0774
new link said. 

> fansub information : from old menome.in.th database. you can find it at [menome data dump](https://archive.org/details/menome)

**Comment by pureexe** [comment]
for indent, i used [nodejs](http://Nodejs.org/). and use `JSON.stringify(data,null,2)`. 
How can i build JSON for this project?

**Comment by FansubDB** [comment]
>it's co-sub NTCS-Fs&DJ-FS on EP1 and EP2 but after that, it was changed to NTCS-Fs&Ayumu-FS

On this situation I sometimes put status `dropped` for NTCS-Fs&DJ-FS and status `release` for NTCS-Fs&Ayumu-FS. Same for 

> from menome data Emergency-FS was alone release until EP.6 and since EP.7 is co-sub with DW

**Comment by FansubDB** [comment]
>for indent, i used nodejs. and use JSON.stringify(data,null,2).
How can i build JSON for this project?

I build JSON using jsoneditoronline and JSONLint (see this [help](https://github.com/FansubDB/fansubdb.github.io/blob/master/CONTRIBUTING.md#the-short-version-modifying-a-json-file))

If you have an external software to help you building JSON file, `JSON.stringify(data,null, "\t")` seems to be more correct.

**Comment by FansubDB** [comment]
Can I merge the PR? Or do you want to change some things?

File: th/2013/spring.json (hunk 2 above)
Comment (pureexe) [inline th/2013/spring.json]: solved on 44b40ca

**Comment by pureexe** [comment]
Ok. I'm ready to merge.

**Pull request merged**